### PR TITLE
Immediate string base for right argument.

### PR DESCRIPTION
**Pull request by MariasStory**
Test if the immediate string base is implemented properly.
In this test only right argument should be changed to string.

### CODE REVIEW

**Comment by Maijin** [comment]
If the test currently doesn't pass you must add `BROKEN=1`

**Comment by MariasStory** [comment]
Write where?

**Comment by Maijin** [comment]
https://github.com/radare/radare2-regressions/blob/master/t.formats/elf/obj#L9

**Comment by Maijin** [comment]
Use the search functionality in github to find examples.

File: t/cmd_ahi [hunk 1]
```diff
@@ -245,3 +245,16 @@ EXPECT='            0x100001000      c7055f440000.  mov dword [rip + 0x445f], 0
...
+NAME='ahi s'
+CMDS='e asm.bits=64
+e asm.arch=x86
+wx c74424685265
+pi 1
+ahi s
+pi 1
+'
+EXPECT='mov dword [rsp + 0x68], 0x6c896552
+mov dword [rsp + 0x68], 'Read'
```
Comment (radare) [inline t/cmd_ahi]: single quote inside a single quote string? this is wrong

**Comment by MariasStory** [comment]
Fixed single quote in single quote problem. Thanks @radare .

**Pull request merged**